base/float.jl

```diff
@@ -703,13 +703,23 @@ end
     realmin() = realmin(Float64)
     realmax() = realmax(Float64)
 
-    eps(x::AbstractFloat) = isfinite(x) ? abs(x) >= realmin(x) ? ldexp(eps(typeof(x)),exponent(x)) : nextfloat(zero(x)) : oftype(x,NaN)
-    eps(::Type{Float16}) = $(box(Float16,unbox(UInt16,0x1400)))
-    eps(::Type{Float32}) = $(box(Float32,unbox(UInt32,0x34000000)))
-    eps(::Type{Float64}) = $(box(Float64,unbox(UInt64,0x3cb0000000000000)))
```

Comment:
I would   leave the hard coded version for these, otherwise I think it will recompute these quantities every time.